firmware/tunerstudio/rusefi.input

```diff
@@ -1920,12 +1920,14 @@ cmd_set_engine_type_default					= "w\x00\x31\x00\x00"
 
 ;			Sensors->MAP sensor
 	dialog = mapSensorAnalog, "MAP sensor", yAxis		
-		field = "MAP ADC input", 						map_sensor_hwChannel
-		field = "MAP type",								map_sensor_type
-		field = "MAP value low point",  		        map_sensor_lowValue
-		field = "MAP voltage low point", 	 		    mapLowValueVoltage
-		field = "MAP value high point",               	map_sensor_highValue
-		field = "MAP voltage high value",               mapHighValueVoltage
+		field = "MAP Sensor Input", 				map_sensor_hwChannel
+		field = "MAP Type",					    	map_sensor_type
+		field = "Linearization Points"
+		field = "MAP Kpa Low",  		            map_sensor_lowValue, {map_sensor_type == 0}
```

Comment:
can you share a snip/screenshot of how this looks?

Reply:
Left some screenshots in another message.